### ISSUE TITLE
nsurlsessiond traffic not captured in Proxyman

### ISSUE THREAD

**Issue by dbyler** [ProxymanApp/Proxyman#1059]
### Proxyman version? (Ex. Proxyman 1.4.3)

Version 2.34.1 (23410)

### macOS Version? (Ex. mac 10.14)

macOS 12.0.1 (21A559)

### Steps to reproduce 

- iOS app running on M1 mac initiates a web request; nsurlsessiond is the application that actually makes the request (as reported by Little Snitch)
- Look for nsurlsessiond in the Proxyman Apps list, or the domain in the Proxyman Domains list

### Expected behavior

1. nsurlsessiond appears as a source of traffic that Proxyman can inspect. But it's missing
2. The domain appears as a domain that Proxyman can inspect. But it's missing, too

**Comment by NghiaTranUIT** [comment]
It means that this traffic doesn't go through System HTTP Proxy at port 9090, where Proxyman is located.

Is it an iOS app that runs on iOS Simulator?

**Comment by dbyler** [comment]
No, it's an iPad app downloaded from the App Store

**Comment by NghiaTranUIT** [comment]
Thanks. Can you verify that Proxyman can capture other HTTP/HTTPS Traffic from your Mac, but not from this iOS app?

For instance, open Google.com on Safari/Chrome from your Mac and see if you can see it on the Proxyman app.

From what I understand, your iPad app (run via Mac) is a Catalyst app, so the traffic might not go through the Proxy.

**Comment by dbyler** [comment]
Yes, I just tried testing a different iPad app downloaded from the App Store, and I was able to capture traffic from that.

**Comment by NghiaTranUIT** [comment]
Sorry, but it's not Proxyman fault. Proxyman could not capture your traffic because your iOS app (run on M1) doesn't respect the System HTTP Proxy (IP = 127.0.0.1, port 9090).

From what I google, there is no solution to capture it unless you're using this on an iPad/iPhone. Then, you can follow this [tutorial](https://docs.proxyman.io/debug-devices/ios-device) to capture it. It will 100% work 👍

**Comment by dbyler** [comment]
Got it, that's disappointing but make sense. Thanks for the quick investigation (and the excellent product), Nghia!
